### ISSUE TITLE
ImportError: cannot import name 'Document'

### ISSUE THREAD

**Issue by aaronn** [sabricot/django-elasticsearch-dsl#187]
After installing the package with the command: `pip install https://github.com/sabricot/django-elasticsearch-dsl/archive/6.4.1.tar.gz`

I get this error message when following the readme:

```
ImportError: cannot import name 'Document'
```

**Comment by PabloCastellano** [comment]
Try:

```
from django_elasticsearch_dsl.documents import Document
```

**Comment by awwester** [comment]
I'm getting this error as well, new install today. I get the error with `from django_elasticsearch_dsl.documents import Document` as well.

**Comment by josh-stableprice** [comment]
I've just taken a look at the released version of this on PyPi and it looks like it's a rebadged version 5.x.x
@safwanrahman @sabricot any chance one of you are able to fix this?

**Comment by josh-stableprice** [comment]
Also the readme is giving the wrong setup instructions as Document has been aliased to DocType

**Comment by safwanrahman** [comment]
This should be fixed in `6.4.2`
